docs/standard/datetime/system-text-json-support.md

```diff
@@ -186,6 +186,8 @@ The following levels of granularity are defined for parsing:
     3. "yyyy'-'MM'-'dd'T'HH':'mm':'ss('+'/'-')HH':'mm"
     4. "yyyy'-'MM'-'dd'T'HH':'mm':'ss'.'FFFFFFF('+'/'-')HH':'mm"
 
+    This level of granularity is compliant with [RFC 3339](https://tools.ietf.org/html/rfc3339#section-5.6), a widely adopted profile of ISO 8601 used for interchanging date and time information.
```

Comment:
How will this appear in the final doc? Don't have it indented.

```suggestion
This level of granularity is compliant with [Internet Date/Time Format from RFC 3339](https://tools.ietf.org/html/rfc3339#section-5.6), a widely adopted profile of ISO 8601 used for interchanging date and time information.
```

![image](https://user-images.githubusercontent.com/6527137/97512712-73bbec00-1947-11eb-9987-9d569a3a31ce.png)

Reply:
The note only applies to granularity no `5.` Per the examples section of RFC 3339 (https://tools.ietf.org/html/rfc3339#section-5.8), it is the full date/time format that is compliant, not the other levels of granularity. I think the current indentation makes sense.

Reply:
I see. Isn't 4 also compliant?

Reply:
No, 4  is not compliant because it does not contain a `time-second` ("ss") section.